README.md

```diff
@@ -444,6 +445,24 @@ Note: For modules use the [@import](http://clang.llvm.org/docs/Modules.html#usin
 #import "NYTUserView.h"
 ```
 
+## Protocols
+
+In a [delegate or data source protocol](https://developer.apple.com/library/ios/documentation/General/Conceptual/CocoaEncyclopedia/DelegatesandDataSources/DelegatesandDataSources.html), the first parameter to each method should be the object sending the message.
+
+This helps disambiguate in cases when an object is the delegate for multiple similarly-typed objects, and it helps clarify intent to readers of a class implementing these delegate methods.
+
+**For example:**
+
+```objc
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath;
```

Comment:
> That was rambly, but hopefully made some sense.

Nah, it made sense.

> I don’t know how much it matters, or how much you care, but it might be nice to make this a non-UIKit example if we want to stick with that pattern.

In this case, given that this example could be a little more complex (and as we've seen from discussion, the wording around the rule still isn't quite perfect), I thought it might be nice to use an example people are already familiar with. But I do not feel strongly about it.